pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -176,6 +183,12 @@ EOF
     Prefix = ..
     EOF
     paxmark m $out/libexec/QtWebEngineProcess
+  '' + ''
```

Comment:
@Mic92 What do you think about this? Should it also be darwin only? Currently the build succeeds incorrectly on darwin, but it might also on linux: https://hydra.nixos.org/build/73282201/log

Reply:
Should be ok.